Firestore/core/src/firebase/firestore/local/leveldb_migrations.cc

```diff
@@ -110,6 +113,71 @@ void ClearQueryCache(leveldb::DB* db) {
   transaction.Commit();
 }
 
+void RemoveMutationDocuments(LevelDbTransaction *transaction,
```

Comment:
The pointer symbol should bind the other way in C++ (i.e. should be `LevelDbTransaction* transaction`).

Also some comments about what these functions do would be helpful.

Reply:
Done.